store/sqlstore/post_store.go

```diff
@@ -2272,6 +2272,20 @@ func (s *SqlPostStore) AnalyticsPostCount(options *model.PostCountOptions) (int6
 		query = query.Where(sq.Eq{"p.DeleteAt": 0})
 	}
 
+	if options.ExcludeSystemPosts {
+		query = query.Where("p.Type NOT LIKE 'system_%'")
+	}
```

Comment:
Non blocker, should this check be if `p.Type` is `""`? That's the type for "normal" user posts. There are post types that are not of the format too `system_%`

Reply:
In here I explicitly want to exclude SystemPosts only, if there are other type of posts added in the future, then those posts _should be included_ unless explicitly asked to exclude them.